### ISSUE TITLE
V key for showing debug panel is not working

### ISSUE THREAD

**Issue by devnewton** [melonjs/boilerplate#23]
The S registered key for showing debug panel is not working:

me.plugin.register.defer(this, me.debug.Panel, "debug", me.input.KEY.V);

The debug panel suggest S to show and H to hide: S do nothing, H make rendering to stop completly.

Not sure if related, but sometimes the Melonjs logo is not draw.

**Comment by hansel-no-kioku** [comment]
Maybe, keepalive should be set true when hide the debug panel.
```javascript
var DebugPanel = me.Renderable.extend({
    ...
    hide : function () {
        if (this.visible) {
            ...
            me.game.world.removeChild(this, true);  // Add true
            ...
        }
    },
    ...
});
```

**Comment by obiot** [comment]
better later than never :P
https://github.com/melonjs/melonJS/commit/d79f90e1c6eac40f52d4c667d7fa7044018f42df

there was a bug (not sure if a bug or a regression) in the current 4.0.0 version that was preventing the pointer event to work once the panel was hidden than made visible again (only works after hiding/showing one more time). It's fixed in the current master (4.1.0) version
